ironfish-cli/src/commands/airdrop/constants.ts

```diff
@@ -0,0 +1,5 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+export const AIRDROP_NOTES_IN_BLOCK = 600
+export const FEE_ORE_PER_AIRDROP = 10n
```

Comment:
Might be slightly clearer to call this `FEE_ORE_PER_NOTE` but I'm not too concerned with it